consumer-test/testCases/s4sdk/consumer-test-neo.yml

```diff
@@ -1,7 +1,7 @@
 # Test case configuration
 referenceAppRepo:
   url: "https://github.com/piper-validation/cloud-s4-sdk-book"
-  branch: "consumer-test-neo"
+  branch: "consumer-test-neo-gpp"
```

Comment:
Left-over from testing, I assume?

Reply:
Ah, PR is still a draft. Was assuming it was ready for review.

Reply:
It may well be that it isn't possible to make these changes without switching to new branches in the test repo. Because the changes would have to be synchronized across two repos. It is however possible, to merge the new branches into the old ones after merging this PR, and then switching back to the old branches in a second PR here.

Reply:
No, as there is no other change in this PR other than throwing out the sdk library, it should be possible to use the branches directly.